worker/handler/handler.go

```diff
@@ -432,6 +509,13 @@ func (h *Handler) nuke() {
 	if err := h.sandbox.Remove(); err != nil {
 		log.Printf("failed to remove sandbox :: %v", err.Error())
 	}
+
+	// a sandbox has been destroyed, attempts to wake up blocked
+	// sandbox creation requests, if any.
+	hms := h.hm.hms
+	if getMemUsage() < hms.sbHardLimit {
```

Comment:
Since we have to poll (since there is no other way to be alerted when a process relinquishes memory), this kind of signaling is an optimization, right?  As such, have we done perf experiments justifying this optimization?

Reply:
Got rid of this in the new commit.